install.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/sh
 
-RNS_DIR="./rns-dev"
-BASE_DIR=$(pwd)
+if [[ $# -eq 0 ]] ; then
+    echo 'Please specify the services to install. Services available: "rns" "storage".'
+    exit 1
+fi
+
+for i in "$@"
+do
+   if [ $i == "rns" ] || [ $i == "storage" ] ;  then
+        SERVICE_DIR="./"$i"-dev"
+        BASE_DIR=$(pwd)
+   
+        cd "$SERVICE_DIR"
+        rm -rf node_modules package-lock.json
+        npm i
```

Comment:
I guess we should actually make this `npm ci`
```suggestion
        npm ci
```

Reply:
It didnt work due to a requirement on the package-lock.json lockfile version. I will leave it as-is for now as it regenerates the package-lock.json anyway.   

`npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.`